.github/workflows/release.yml

```diff
@@ -0,0 +1,52 @@
+name: Release SDK, devpack, Gradle plugin
+
+on:
+  workflow_dispatch:
+    inputs:
+      snapshot:
+        description: 'Publish snapshot?'
+        required: false
+        default: 'false'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Set up JDK
+        uses: actions/setup-java@v4
+        with:
+          distribution: 'adopt'
+          java-version: '8'
+          cache: 'gradle'
+      - name: Login to GitHub Container Registry # Required for integration tests
+        uses: docker/login-action@v1
+        with:
+          registry: ghcr.io
+          username: ${{ secrets.CR_PAT_USERNAME }}
+          password: ${{ secrets.CR_PAT }}
+      - name: Unit Tests & Integration Tests
+        run: ./gradlew --info --warning-mode=all clean test integrationTest
+      - name: Release
+        if: github.event.inputs.snapshot != 'true'
+        env:
+          ORG_GRADLE_PROJECT_signingKey: ${{ secrets.GPG_KEY_ARMORED }}
+          ORG_GRADLE_PROJECT_signingPassword: ${{ secrets.GPG_KEY_PASSWORD }}
+          ORG_GRADLE_PROJECT_sonatypeUsername: ${{ secrets.SONATYPE_USERNAME }}
+          ORG_GRADLE_PROJECT_sonatypePassword: ${{ secrets.SONATYPE_PASSWORD }}
+          GRADLE_PUBLISH_KEY: ${{ secrets.GRADLE_PUBLISH_KEY }}
+          GRADLE_PUBLISH_SECRET: ${{ secrets.GRADLE_PUBLISH_SECRET }}
+        run: | # If the neofs module is to be published in the future, remove the -x :neofs:publishToSonatype
+          ./gradlew --info -x test -x :gradle-plugin:publishToSonatype -x :neofs:publishToSonatype publishToSonatype closeAndReleaseSonatypeStagingRepository
+          ./gradlew :gradle-plugin:publishPlugin -Pgradle.publish.key=$GRADLE_PUBLISH_KEY -Pgradle.publish.secret=$GRADLE_PUBLISH_SECRET
+      - name: Release Snapshot
+        if: github.event.inputs.snapshot == 'true'
+        env:
+          ORG_GRADLE_PROJECT_signingKey: ${{ secrets.GPG_KEY_ARMORED }}
+          ORG_GRADLE_PROJECT_signingPassword: ${{ secrets.GPG_KEY_PASSWORD }}
+          ORG_GRADLE_PROJECT_sonatypeUsername: ${{ secrets.SONATYPE_USERNAME }}
+          ORG_GRADLE_PROJECT_sonatypePassword: ${{ secrets.SONATYPE_PASSWORD }}
+        run: | # If you want to release a snapshot of gradle-plugin, remove the -x :gradle-plugin:publishToSonatype
```

Comment:
We should always release the `gradle-plugin` to the SNAPSHOT, IMHO. This makes development easier, and to test things.

Reply:
So, it is possible to fetch the gradle-plugin from the Sonatype snapshot repo artifacts?
I never did that. I always published to maven local and tested plugin changes from there.

Reply:
Maybe, would you like to document this to `./gradle-plugin/README.md`?

Reply:
yes, will do 👍

Reply:
Ok done. All instructions on how to release locally and release snapshots are now in the DEVELOPMENT.md file.

Reply:
Great! Checking.